Applications/Games/Space Engineers/Steam/script.js

```diff
@@ -2,26 +2,24 @@ const SteamScript = include("engines.wine.quick_script.steam_script");
 include("engines.wine.verbs.dotnet472");
 include("engines.wine.verbs.vcrun2017");
 include("engines.wine.verbs.dxvk");
-include("engines.wine.verbs.faudio");
 include("engines.wine.plugins.override_dll");
 
 new SteamScript()
     .name("Space Engineers")
     .editor("Keen Software House")
     .author("Zemogiter")
-    .appId("244850")
-    .wineVersion("4.14")
+    .appId(244850)
+    .wineVersion("4.15")
     .wineDistribution("upstream")
     .wineArchitecture("amd64")
     .preInstall(function (wine, wizard) {
         wine.dotnet472();
         wine.vcrun2017();
         wine.DXVK();
-        wine.faudio();
         wine.overrideDLL()
-            .set("native, builtin", ["msvcr120", "xaudio2_0", "xaudio2_1", "xaudio2_2", "xaudio2_3", "xaudio2_4", "xaudio2_5", "xaudio2_6", "xaudio2_7", "xaudio2_8", "xaudio2_9", "x3daudio1_3", "x3daudio1_4", "x3daudio1_5", "x3daudio1_6", "x3daudio1_7"])
+            .set("native, builtin", ["msvcr120"])
             .do();
         wizard.message(tr("You have to install libjpeg62 and libjpeg62-dev or else the thumbnails in New Game menu will be dispalyed as magenta rectangles."));
-        wizard.message(tr("Due to JIT compiler issues and the way this game uses multithreating, there are audio stutters. This script will attempt to minimize them but you might also have to enter the alsoft-conf command in terminal and set sample depth to 32bit float and period size to 2048."));
+        wizard.message(tr("Due to JIT compiler issues and the way this game uses multithreating, there are audio stutters. To minimize those, download the libFAudio package from this PPA:\nhttps://launchpad.net/~cybermax-dexter/+archive/ubuntu/sdl2-backport"));
```

Comment:
I don't think we can do this: we cannot generally assume that our users are using Ubuntu.

Reply:
Can we can use the `operatingSystem Fetcher` to detect if user runs an distribution that isn't Debian-based? We could display a more fitting message then.

Reply:
This is only used for Linux/OSX. I don't think we want distribution specific code.
@madoar @ImperatorS79 any ideas how we could handle this?

Reply:
We may be lucky and one of the system properties in https://docs.oracle.com/javase/tutorial/essential/environment/sysprop.html contains the necessary information. If not we need to use additional libraries to fetch this information in a platform independent manner. I can't say yet if there is a library that also works on android if we want to target that system later on.

Reply:
I would rather propose not to focus on any specific distribution and keep the message more general in cases like this. So e.g. "If you experience audio stutter, you might want to use libFAudio.". Or maybe even better: a link to WineHQ bug report.

Reply:
@plata there isn't a bug report on that the entire WineHQ page on that game is out of date.

Reply:
I could but I'm away from my main PC and the laptop I use isn't powerful enough to run this game. Also my Ubuntu installation on that PC got corrupted somehow and the display loses signal before I get to the login screen.